pkg/netpol/eval/internal/k8s/peer.go

```
@@ -91,7 +85,7 @@ func (p *WorkloadPeer) Kind() string {
 }
 
 func (p *WorkloadPeer) String() string {
```

is this function valid if `p` is representative peer?

---

it is never used for representative peer.

---

the code will be more robust if valid for this case as well, or if issue an error in that case as not supported for this function.

---

I've widen the case of fake-pod for "representative-peer" too.
it is simply not used since we don't care of the String representation of a representative peer; we just need its labels. 
so I don't think should raise an error too.

---

ok, better add a comment that it is not expected to be used for  representative peer.

---

done